src/core/requests/send_contact.rs

```diff
@@ -1,7 +1,142 @@
-use crate::core::requests::RequestContext;
-//TODO:: need implementation
+use crate::core::network;
+use crate::core::requests::{
+    ChatId, Request, RequestContext, RequestFuture, ResponseResult,
+};
+use crate::core::types::{Message, ReplyMarkup};
+
+///Use this method to send phone contacts. On success, the sent Message is
+/// returned.
```

Comment:
I think this comment (`... Message is returned`) is redundant (you can just look into the implementation of Request for this type)